src/Controls/src/Core/Button/Button.Mapper.cs

```diff
@@ -19,6 +19,9 @@ public partial class Button
 			ButtonHandler.Mapper.ReplaceMapping<Button, IButtonHandler>(nameof(Padding), MapPadding);
 			ButtonHandler.Mapper.ReplaceMapping<Button, IButtonHandler>(nameof(BorderWidth), MapBorderWidth);
 #endif
+#if ANDROID
+			ButtonHandler.Mapper.ReplaceMapping<Button, IButtonHandler>(nameof(PlatformConfiguration.AndroidSpecific.Button.RippleColorProperty), MapRippleColor);
```

Comment:
This will result in a mapping of `"RippleColorProperty"`, this should be a string or even just:

```suggestion
			ButtonHandler.Mapper.ReplaceMapping<Button, IButtonHandler>(PlatformConfiguration.AndroidSpecific.Button.RippleColorProperty.PropertyName, MapRippleColor);
```